charts/azuremonitor-containers/templates/omsagent-daemonset.yaml

```diff
@@ -103,6 +103,14 @@ spec:
        {{- end }}
        - name: ISTEST
          value: {{ .Values.omsagent.ISTEST | quote }}
+       {{ if .Values.omsagent.arcasettings.isArcACluster }}
+       - name: IS_ARCA_CLUSTER
+         value: {{ .Values.omsagent.arcasettings.isArcACluster | quote }}
+       {{- if ne .Values.omsagent.arcasettings.metricsEndpoint "<your_metrics_endpoint>" }}
```

Comment:
since metrics endpoint is set by the user, so will have to validate this endpoint is in valid format or not.

Reply:
In practice, this endpoint will be set by Arc-A control plane rather than user, since the endpoint is unique on one Arc-A cluster. We'd like to give it a default value in values.yml before push helm chart into on-prem registry.